spec/unit/action/puppetfile/install_spec.rb

```diff
@@ -51,14 +58,16 @@ def installer(opts = {}, argv = [], settings = {})
 
   describe "purging" do
     before do
-      allow(puppetfile).to receive(:modules).and_return([])
+      allow(loader).to receive(:load).and_return({
+        modules:             [],
+        desired_contents:    [ 'root/foo' ],
+        managed_directories: [ 'root' ],
+        purge_exclusions:    [ 'root/**/**.rb' ]
+      })
```

Comment:
I don't really mind, but is there a reason this (and the test below) have these in `before` blocks, even though there's only one test case?

Reply:
No, it's a weird pattern this whole test file used. I just made it more obvious by making the single line stub be six lines. I've updated the whole file to pull them into the `it` blocks when possible.